### ISSUE TITLE
Draugir token (200457) is incorrectly marked as unreleased

### ISSUE THREAD

**Issue by jamieadkins95** [GwentCommunityDevelopers/gwent-data#14]
Draugir is a token of Draug, but has not been marked as released. Will look into this today.

**Comment by jamieadkins95** [comment]
In GwentCardAbilities.xml there is a new way of defining what cards are spawned"

```
      <SpawnEntityList>
        <SpawnEntity>
          <TemplateId V="200457"/>
          <Side V="Same"/>
          <Location V="Graveyard"/>
          <Occupied V="Ignore"/>
          <NumberOfCopies IntType="ConstantValue" V="1"/>
        </SpawnEntity>
      </SpawnEntityList>
```

Draugir example:

```
    <!--- - - - - - - - - - - - - -  Draug_SpawnDraugir - - - - - - - - - - - - - - -->
    <Ability id="Draug_SpawnDraugir" groupid="NONE" type="SpawnEntities">
      <Priority IntType="ConstantValue" V="400"/>
      <NumOfTimes IntType="ConstantValue" V="1"/>
      <SetTriggerInvalid V="False"/>
      <TriggeredIfDead V="False"/>
      <OnlyOneOfKindPerTrigger V="False"/>
      <UserInteraction V="None"/>
      <WillAffect V="Picked"/>
      <AbilityTriggerConditions>
        <Trigger type="OnCardsAffectedByAbility">
          <TriggerOwner V="Self"/>
          <ActiveLocation V="Active"/>
          <IgnoreSilence V="False"/>
          <SourceAbility V="">
            <Item v="Draug_FilterEnemy"/>
          </SourceAbility>
          <FullCheckConditions CheckerList="AliveCards"/>
          <FullCheckConditionsCardRelated CheckerList="TriggerCards">
            <CardCountCondition>
              <CardCount IntType="CardCount" V="">
                <CardCountConditions CheckerList="AliveCards">
                  <BoardLocationCondition>
                    <CheckAgainst V="SourceCard"/>
                    <BoardLocation V="Graveyard"/>
                  </BoardLocationCondition>
                  <LoyaltyCondition IsNegated="True">
                    <CheckAgainst V="SourceCard"/>
                  </LoyaltyCondition>
                  <GroupCondition IsNegated="True">
                    <Group V="Gold"/>
                  </GroupCondition>
                </CardCountConditions>
              </CardCount>
              <CheckType V="GreaterOrEqual"/>
              <TargetValue IntType="ConstantValue" V="1"/>
            </CardCountCondition>
          </FullCheckConditionsCardRelated>
        </Trigger>
      </AbilityTriggerConditions>
      <Conditions CheckerList="AliveCards"/>
      <SpawnEntityList>
        <SpawnEntity>
          <TemplateId V="200457"/>
          <Side V="Opposite"/>
          <Location V="Graveyard"/>
          <Occupied V="Ignore"/>
          <NumberOfCopies IntType="ConstantValue" V="1"/>
        </SpawnEntity>
      </SpawnEntityList>
      <BanishEntityList/>
    </Ability>
    <!--- - - - - - - - - - - - - -  Draug_SpawnDraugirAlly - - - - - - - - - - - - - - -->
    <Ability id="Draug_SpawnDraugirAlly" groupid="NONE" type="SpawnEntities">
      <Priority IntType="ConstantValue" V="399"/>
      <NumOfTimes IntType="ConstantValue" V="1"/>
      <SetTriggerInvalid V="False"/>
      <TriggeredIfDead V="False"/>
      <OnlyOneOfKindPerTrigger V="False"/>
      <UserInteraction V="None"/>
      <WillAffect V="Picked"/>
      <AbilityTriggerConditions>
        <Trigger type="OnCardsAffectedByAbility">
          <TriggerOwner V="Self"/>
          <ActiveLocation V="Active"/>
          <IgnoreSilence V="False"/>
          <SourceAbility V="">
            <Item v="Draug_FilterEnemy"/>
          </SourceAbility>
          <FullCheckConditions CheckerList="AliveCards"/>
          <FullCheckConditionsCardRelated CheckerList="TriggerCards">
            <CardCountCondition>
              <CardCount IntType="CardCount" V="">
                <CardCountConditions CheckerList="AliveCards">
                  <BoardLocationCondition>
                    <CheckAgainst V="SourceCard"/>
                    <BoardLocation V="Graveyard"/>
                  </BoardLocationCondition>
                  <LoyaltyCondition>
                    <CheckAgainst V="SourceCard"/>
                  </LoyaltyCondition>
                  <GroupCondition IsNegated="True">
                    <Group V="Gold"/>
                  </GroupCondition>
                </CardCountConditions>
              </CardCount>
              <CheckType V="GreaterOrEqual"/>
              <TargetValue IntType="ConstantValue" V="1"/>
            </CardCountCondition>
          </FullCheckConditionsCardRelated>
        </Trigger>
      </AbilityTriggerConditions>
      <Conditions CheckerList="AliveCards"/>
      <SpawnEntityList>
        <SpawnEntity>
          <TemplateId V="200457"/>
          <Side V="Same"/>
          <Location V="Graveyard"/>
          <Occupied V="Ignore"/>
          <NumberOfCopies IntType="ConstantValue" V="1"/>
        </SpawnEntity>
      </SpawnEntityList>
      <BanishEntityList/>
    </Ability>

```

**Comment by jamieadkins95** [comment]
Fixed in #15
